src/main.ts

```diff
@@ -375,18 +389,27 @@ async function run(): Promise<void> {
       token: apiToken,
     });
   }
+
+  //
 }
 
 const {headRef, headSha} = getGithubHeadRefInfo();
 
 const transaction = Sentry.startTransaction({
   op: shouldSaveOnly !== 'false' ? 'save snapshots' : 'run',
-  name: 'visual snapshot',
+  // This is the title field in Discover
+  name: shouldSaveOnly !== 'false' ? 'save snapshots' : 'run',
```

Comment:
This fixes all transactions being named the same.

Reply:
I don't remember, do we need both op and name or can only using name be sufficient?

Reply:
Using the name is sufficient.